<ui version="4.0" >
 <class>colorBarDialog</class>
 <widget class="QDialog" name="colorBarDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Color bar</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../ElmerGUI.qrc" >
    <normaloff>:/icons/Mesh3D.ico</normaloff>:/icons/Mesh3D.ico</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="colorGroup" >
     <property name="title" >
      <string>Control</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="colorLabel" >
          <property name="text" >
           <string>Color map:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="colorCombo" >
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="layoutGroup" >
     <property name="title" >
      <string>Layout</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QRadioButton" name="horizontalRButton" >
          <property name="text" >
           <string>Horizontal</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QLabel" name="labelsLabel" >
          <property name="text" >
           <string>Labels:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" >
         <widget class="QSpinBox" name="labelsSpin" >
          <property name="value" >
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QRadioButton" name="verticalRButton" >
          <property name="text" >
           <string>Vertical</string>
          </property>
          <property name="checked" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QLabel" name="widthLabel" >
          <property name="text" >
           <string>Width:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3" >
         <widget class="QLineEdit" name="widthEdit" >
          <property name="text" >
           <string>0.1</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QCheckBox" name="annotateBox" >
          <property name="text" >
           <string>Annotate field name</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="2" >
         <widget class="QLabel" name="heightLabel" >
          <property name="text" >
           <string>Length:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3" >
         <widget class="QLineEdit" name="heightEdit" >
          <property name="text" >
           <string>0.9</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton" >
       <property name="text" >
        <string>Apply</string>
       </property>
       <property name="icon" >
        <iconset>
         <normaloff/>
        </iconset>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
       <property name="icon" >
        <iconset resource="../ElmerGUI.qrc" >
         <normaloff>:/icons/dialog-close.png</normaloff>:/icons/dialog-close.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>Ok</string>
       </property>
       <property name="icon" >
        <iconset resource="../ElmerGUI.qrc" >
         <normaloff>:/icons/dialog-ok-apply.png</normaloff>:/icons/dialog-ok-apply.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ElmerGUI.qrc" />
  <include location="../ElmerGUI.qrc" />
 </resources>
 <connections/>
</ui>
